src/pages/settings/bank-accounts/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export { default as BankAccounts } from './index/BankAccounts';
+export { default as EditBankAccount } from './edit/Edit';
+export { default as CreateBankAccount } from './create/Create';
+export { default as BankAccountDetails } from './show/BankAccountDetails';
```

Comment:
No need for this. Use named exports to avoid `index.ts`.

Reply:
> No need for this. Use named exports to avoid `index.ts`.

I removed default exports, but I should keep this file for routing.

![image](https://user-images.githubusercontent.com/51542191/201727238-ce0ff029-62d6-4e11-a6e8-6c8922ed57ee.png)